llvm/include/llvm/Analysis/VecFuncs.def

```diff
@@ -14,7 +14,7 @@
 
 #if defined(TLI_DEFINE_MASSV_VECFUNCS_NAMES)
 #define TLI_DEFINE_MASSV_VECFUNCS
-#define TLI_DEFINE_VECFUNC(SCAL, VEC, VF) VEC,
+#define TLI_DEFINE_VECFUNC(SCAL, VEC, VF, MANGLN) VEC,
```

Comment:
Now that we've settled on just storing the prefix I suggest `VABI_PREFIX` as a more accurate name.